extension/landing-zone/variables.tf

```diff
@@ -0,0 +1,324 @@
+variable "ibmcloud_api_key" {
+  description = "APIkey that's associated with the account to provision resources to"
+  type        = string
+  sensitive   = true
+}
+
+variable "resource_group" {
+  type        = string
+  description = "An existing resource group name to use for this example, if unset a new resource group will be created"
+  default     = null
+}
+
+variable "region" {
+  description = "The region to which to deploy the VPC"
+  type        = string
+  default     = "au-syd"
+}
+
+variable "prefix" {
+  description = "The prefix that you would like to append to your resources"
+  type        = string
+  default     = "slz-vsi"
+}
+
+variable "vpc_id" {
+  description = "Id of the VPC in which the VSI will be created."
+  type        = string
+  default     = "r026-9c433ed0-69f5-4554-ab35-c86d01ef1b6c"
+}
+
+variable "ssh_key" {
+  type        = string
+  description = "An existing ssh key name to use for this example, if unset a new ssh key will be created"
```

Comment:
Can we be more specific here? 

The name of a public SSH Key which already exists in the deployment region that will be used for VSI creation.

I'm not sure if we should support creating an SSH key in this DA extension. Its not something that would ever be done. However, I think we should also support adding an SSH key. The same way the VSI DA does right now using:

```
variable "ssh_public_key" {
  description = "A public SSH Key for VSI creation which does not already exist in the deployment region. Must be an RSA key with a key size of either 2048 bits or 4096 bits (recommended) - See https://cloud.ibm.com/docs/vpc?topic=vpc-ssh-keys. To use an existing key, enter a value for the variable 'existing_ssh_key_name' instead."
  type        = string
  default     = null
  validation {
    error_message = "Public SSH Key must be a valid ssh rsa public key."
    condition     = var.ssh_public_key == null || can(regex("ssh-rsa AAAA[0-9A-Za-z+/]+[=]{0,3} ?([^@]+@[^@]+)?", var.ssh_public_key))
  }
}
```